### DISCUSSION TITLE
Output values of pole figures at specific locations

### DISCUSSION THREAD

**Discussion by creuzige** [mtex-toolbox/mtex#497] [Ask Anything] [answered]
I am trying to export the pole figure intensity values at specific locations on the pole figure (like along the xvector, zvector or similar).  But I wasn't able to find a command to do so in the function references list.  I dimly recall the 'get' function in versions 3 and 4 allowed this.  Is there a currently a command that I'm just missing?  If not, could it be added?

**Comment by ralfHielscher** [comment]
Assume you have a single pole figure you can do
```
pf(xvector).intensities
```
this will give you the intensity of the measurement point closest to xvector

Ralf.




********************************************************************
Ralf Hielscher                   Tel: +371-531-38556
Fakultät für Mathematik               +371-531-22200 (Sekr.)
Technische Universität Chemnitz  Fax: +371-531-22109
Reichenhainer Str. 39            E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz                 http://www.tu-chemnitz.de/~rahi
********************************************************************

**Comment by creuzige** [comment, marked as answer]
Thanks!  It also looks like you can send an array of vectors, and if you have multiple pole figures a short for loop will work. Example below works with several pole figures in a single class (pf), and two vectors (x and z axis).

```
for i=1:length(pf.c)
    pf_temp=pf{i};
    pf_temp([xvector,zvector]).intensities
end
```
